data/serializers.py

```diff
@@ -98,10 +97,16 @@ class Meta:
         fields = ('id', 'user', 'token', 'endpoint')
 
 
+class ReadOnlyDDSUserCredSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = DDSUserCredential
+        resource_name = 'dds-user-credentials'
+        fields = ('id', 'user', 'endpoint')
```

Comment:
I see, so the `ReadOnlyDDSUserCredSerializer` only serializes the id, user and endpoint fields - not the actual credentials in `DDSUserCredential`